File: conditionals/old.py
# David Garcia Conditionals Old enough Py
<<<<<<< HEAD
age = int(input(("How old are you:\n")))
if age >= 16:
    print("You are able to get a drivers license")
elif age >= 20:
    print("You are able to go to school, you are able to vote, you are able to get a learners permit, and you are able to get a drivers license")
elif age== 15:
    print("You are able to get a learners permit")
elif age == 0:
    print("bro was just born.")
elif age >= 5:
    print("You are able to go to school")
else:
    print("You are not old enough to do anything")
=======
age = int(input("How old are you:\n"))

if age >= 21:
    print("You can vote, go to schoool, get a learners permit, and a drivers license.")
elif age >= 16 and age <= 21:
    print("You can get a driver's license, get a learners permit, and go to school")
elif age >= 15 and age < 16:
    print("You can get a learner's permit, and go to school")
elif age >= 5 and age <= 1:
    print("You can go to school.")
elif age == 0:
    print("bro was just born")
else:
    print("You are too young.")
>>>>>>> b3e1d7a211c4930630db9c8c006f62993033595e

File: conditionals/old.c
<<<<<<< HEAD
// David Garcia, Templates C 
#include <stdio.h>
#include <string.h> 
#include <math.h>
float age;

int main(void){
age = int(input(("How old are you:\n")));
    scanf("%d", &age);
    printf("Hello %s welcome to my program!", name);

if (age >= 17){
    printf("You are able to get a drivers license");
}else if (age >= 21){
    printf("You are able to vote");
}else if (age >= 15){
    printf("You are able to get a learners permit");
}else if (age == 0){
    printf("bro was just born.");
}else if (age >4){
    printf("You are able to go to school");
}else{
    printf("You are so young to do something");
}
    return 0;
}
=======
// David Garcia  C old enough
#include <stdio.h>

int main(void) {
    int age;

    printf("How old are you:\n");
    scanf("%d", &age);

    if (age >= 21) {
        printf("You can vote, go to school, get a learner's permit, and a driver's license.\n");
    } else if (age >= 16 && age <= 21) {
        printf("You can get a driver's license, get a learner's permit, and go to school.\n");
    } else if (age >= 15 && age < 16) {
        printf("You can get a learner's permit, and go to school.\n");
    } else if (age >= 5 && age <= 14) {
        printf("You can go to school.\n");
    } else if (age == 0) {
        printf("Bro was just born.\n");
    } else {
        printf("You are too young.\n");
    }

    return 0;
}
>>>>>>> b3e1d7a211c4930630db9c8c006f62993033595e
